core/trino-main/src/test/resources/cert/generate.sh

```diff
@@ -2,6 +2,16 @@
 
 set -eux
 
-openssl req -new -x509 -newkey rsa:4096 -sha256 -nodes -keyout localhost.key -days 3560 -out localhost.crt -config localhost.conf
-cat localhost.crt localhost.key > localhost.pem
+script_dir=$(CDPATH="" cd -- "$(dirname -- "$0")" && pwd)
+outdir=$1
```

Comment:
maybe instead of outdir we could specify the target pem file name?

Reply:
Done!